gflownet/envs/tetris.py

```diff
@@ -75,6 +75,7 @@ def __init__(
         height: int = 20,
         pieces: List = ["I", "J", "L", "O", "S", "T", "Z"],
         rotations: List = [0, 90, 180, 270],
+        flatten: bool = True,
```

Comment:
If we move the flattening from the environment to the policy, then we don't need this.